tests are failing. think you have to remove them or use `softer_assert`.
```
======================================================================
FAIL: test_submit_user_registration (corehq.apps.receiverwrapper.tests.test_submissions.SubmissionTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/commcare-hq-ro/corehq/util/test_utils.py", line 232, in inner
    return helper(*args, **kwargs)
  File "/mnt/commcare-hq-ro/corehq/util/test_utils.py", line 205, in __call__
    call_with_settings(fn_with_pre_and_post, run_config.settings, args, kwargs)
  File "/mnt/commcare-hq-ro/corehq/util/test_utils.py", line 220, in call_with_settings
    fn(*args, **kwargs)
  File "/mnt/commcare-hq-ro/corehq/util/test_utils.py", line 201, in fn_with_pre_and_post
    self.fn(*args, **kwargs)
  File "/mnt/commcare-hq-ro/corehq/apps/receiverwrapper/tests/test_submissions.py", line 97, in test_submit_user_registration
    xmlns='http://openrosa.org/user/registration',
  File "/mnt/commcare-hq-ro/corehq/apps/receiverwrapper/tests/test_submissions.py", line 68, in _test
    self.assertEqual(n_times_saved, 1)
AssertionError: 2 != 1
-------------------- >> begin captured stdout << ---------------------
<function test_submit_user_registration at 0x7f89a38d6230> failed with the following settings:
settings.TESTS_SHOULD_USE_SQL_BACKEND = False
--------------------- >> end captured stdout << ----------------------
-------------------- >> begin captured logging << --------------------
auditcare.middleware: DEBUG: Middleware is running in a test context, disabling monkeypatch
root: ERROR: someone actually called CommCareUser.create_or_update_from_xform
Traceback (most recent call last):
  File "/mnt/commcare-hq-ro/corehq/apps/users/signals.py", line 61, in create_user_from_commcare_registration
    CommCareUser.create_or_update_from_xform(xform)
  File "/mnt/commcare-hq-ro/corehq/apps/users/models.py", line 1442, in create_or_update_from_xform
    _assert(False, 'someone actually called CommCareUser.create_or_update_from_xform')
  File "/mnt/commcare-hq-ro/corehq/util/soft_assert/core.py", line 33, in __call__
    return self._call(assertion, msg, obj)
  File "/mnt/commcare-hq-ro/corehq/util/soft_assert/core.py", line 47, in _call
    raise AssertionError(msg)
AssertionError: someone actually called CommCareUser.create_or_update_from_xform
root: ERROR: Receiver app: problem sending post-save signal <function create_user_from_commcare_registration at 0x7f89b00b1140> for xform d8d7ebf232cb7bce926e11e214f06024: AssertionError: someone actually called CommCareUser.create_or_update_from_xform
--------------------- >> end captured logging << ---------------------
```

--- a/corehq/apps/receiverwrapper/tests/test_submissions.py
+++ b/corehq/apps/receiverwrapper/tests/test_submissions.py
@@ -6,7 +6,7 @@ from django.test.utils import override_settings
 from corehq.apps.users.models import WebUser
 from corehq.apps.domain.shortcuts import create_domain
 from corehq.apps.receiverwrapper import submit_form_locally
-from corehq.util.test_utils import TestFileMixin
+from corehq.util.test_utils import TestFileMixin, softer_assert
 from django.test.client import Client
 from django.core.urlresolvers import reverse
 import os
@@ -91,6 +91,7 @@ class SubmissionTest(TestCase):
         )
 
     @run_with_all_backends
+    @softer_assert
     def test_submit_user_registration(self):
         self._test(
             form='user_registration.xml',

--- a/corehq/apps/users/tests/create.py
+++ b/corehq/apps/users/tests/create.py
@@ -3,6 +3,7 @@ from django.test import TestCase
 from corehq.apps.users.dbaccessors.all_commcare_users import delete_all_users
 from corehq.apps.users.util import format_username
 from corehq.util.dates import iso_string_to_date
+from corehq.util.test_utils import softer_assert
 from couchforms.models import XFormInstance
 from corehq.apps.users.models import CouchUser, WebUser, CommCareUser
 from dimagi.utils.dates import force_to_datetime
@@ -92,13 +93,16 @@ class CreateTestCase(TestCase):
         django_user = couch_user.get_django_user()
         self.assertEqual(couch_user.user_id, CouchUser.from_django_user(django_user).user_id)
 
+    @softer_assert
     def testCreateUserFromRegistration(self):
         self._runCreateUserFromRegistrationTest()
 
+    @softer_assert
     def testCreateUserFromOldRegistration(self):
         self.xform.xmlns = USER_REGISTRATION_XMLNS_DEPRECATED
         self._runCreateUserFromRegistrationTest()
 
+    @softer_assert
     def testCreateDuplicateUsersFromRegistration(self):
         """
         use case: chw on phone registers a username/password/domain triple somewhere
@@ -115,6 +119,7 @@ class CreateTestCase(TestCase):
         self.assertEqual("test_reg", first_user.username.split("@")[0])
         self.assertEqual("test_reg2", second_user.username.split("@")[0])
 
+    @softer_assert
     def testEditUserFromRegistration(self):
         """
         Edit a user via registration XML
@@ -147,6 +152,7 @@ class CreateTestCase(TestCase):
         self.assertNotEqual(original_django_user.username, updated_django_user.username)
         self.assertNotEqual(original_django_user.password, updated_django_user.password)
 
+    @softer_assert
     def testEditUserFromRegistrationWithConflicts(self):
         original_user, created = CommCareUser.create_or_update_from_xform(self.xform)
         self.assertEqual("test_reg", original_user.username.split("@")[0])

